src/Microsoft.AspNetCore.Mvc.Localization/DependencyInjection/MvcLocalizationMvcBuilderExtensions.cs

```diff
@@ -2,14 +2,15 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using Microsoft.AspNetCore.Mvc.DataAnnotations;
 using Microsoft.AspNetCore.Mvc.Localization.Internal;
 using Microsoft.AspNetCore.Mvc.Razor;
 using Microsoft.Extensions.Localization;
 
 namespace Microsoft.Extensions.DependencyInjection
 {
     /// <summary>
-    /// Extension methods for configuring MVC view localization.
+    /// Extension methods for configuring MVC localization.
```

Comment:
@Eilon we don't have tests confirming these methods add what's expected. Do we need a few?

@kichalla please add `AddMvcLocalization(...)` methods to `MvcLocalizationMvcCoreBuilderExtensions`.

Reply:
Are you referring to the newly added extension? There are unit tests for the `AddMvcViewLocalizationServices` though. I guess we didn't want to add tests for non-Mvc services existence because we would be tight coupling across repos and localization services and its dependencies can change anytime causing our tests to fail

Reply:
There are no unit tests for this class or `MvcLocalizationMvcCoreBuilderExtensions` at all.